runtime/doc/treesitter.txt

```diff
@@ -1055,6 +1055,10 @@ edit({lang})                                     *vim.treesitter.query.edit()*
     Parameters: ~
       • {lang}  (`string?`) language to open the query editor for. If omitted,
                 inferred from the current buffer's filetype.
+      • {opts}  (`table?`) Optional keyword arguments:
+                • {wincmd}? (`string`) command used to create a new window.
```

Comment:
not a blocker, just for reference:

I did a quick search and didn't find existing `wincmd` key for other cases (in lsp/ts/lua/diagnostics). And I think that's because we usually end up exposing this kind of thing as a callback (e.g. `on_list` for `:help vim.lsp.ListOpts`). E.g. this could be something like:

    {on_open}? (func (...): ...)

However that might complicate the implementation since we can't just pass it to the existing `vim.cmd()` call.